editor.planx.uk/src/routes/flow.tsx

```diff
@@ -204,8 +209,12 @@ const routes = compose(
 
   withView(async (req) => {
     const [flow, ...breadcrumbs] = req.params.flow.split(",");
-    const settings: FlowSettings = await getFlowSettings(flow, req.params.team);
+    const { settings, flowMetabaseLink } = await getFlowMetadata(
+      flow,
+      req.params.team,
+    );
     useStore.getState().setFlowSettings(settings);
+    useStore.setState({ flowMetabaseLink });
```

Comment:
You might be able to simplify things here and set both `flowSettings` and `flowMetabaseLink` in the same setter.

Something like `useStore.setState({ flowMetabaseLink, flowSettings });`

Reply:
I'll check this out 👍

Reply:
Fixed here: 7d604531d87fb0a147cedb199731daac885b8a40

I suspect that `.setFlowSettings` can probably be removed and associated files updated to use the altered syntax. 

I might have a go adding that as a commit once I've updated as per the other comments.

Reply:
Yep - sounds good - any chance to rationalise and remove code!

Reply:
If you don't mind @DafyddLlyr I'm going to do this in another branch to avoid muddying things in this PR